docs/concepts/SafeRollout/README.md

```diff
@@ -184,22 +184,15 @@ as available in the target clusters when they meet the criteria we defined.
 We have an agent running in the target cluster to check the status of the resources. We have specific criteria for each 
 of the following resources to determine if they are available or not. Here are the list of resources we support:
 
-#### Deployment
-We only mark a `Deployment` as available when all its pods are running, ready and updated according to the latest spec. 
-
-#### DaemonSet 
-We only mark a `DaemonSet` as available when all its pods are available and updated according to the latest spec on all 
-desired scheduled nodes.
-
-#### StatefulSet
-We only mark a `StatefulSet` as available when all its pods are running, ready and updated according to the latest revision.
+#### Deployment/DaemonSet/StatefulSet
+We only mark them as available if the number of updated replicas equals the number of desired replicas.

```

Comment:
I think replicas are not relevant for Daemonsets since they are meant for running a pod on desired nodes

Reply:
"replica" means the number of pods